schema/mutation.graphql

```diff
@@ -5,71 +5,71 @@ module.exports = `
     login(data: LoginInput!): AuthData!
     refreshToken(refreshToken: String!) : ExtendSession!
     revokeRefreshTokenForUser(userId: String!) : Boolean!
```

Comment:
I had a question here. This mutation is also supposed to be authenticated right?